src/kernel/formulations.jl

```diff
@@ -32,3 +32,69 @@ where λ is a shape parameter.
     λ::P = Param(1.0, bounds=(0.0, 2.0))
 end
 (f::ExponentialKernel)(distance) = exp(-distance / f.λ)
+
+"""
+    GeometricKernel <: KernelFormulation
+
+    GeometricKernel(α)
+
+Probability density function of distance ``d``.
+
+The Geometric kernel has a power-law decrease and is isotropic.
+
+```math
+y = \frac{(α+1)(α+2)}{2 π} (1+d)^α
+```
+
+where α is a shape parameter.
+"""
+@Base.kwdef struct GeometricKernel{P} <: KernelFormulation
+    α::P = Param(1.0, bounds=(-1000.0, 1000.0))
+end
+(f::GeometricKernel)(distance) = (1+distance)^f.α * ((f.α+1)*(f.α+2))/(2*π)
+
+
+"""
+    GaussianKernel <: KernelFormulation
+
+    GaussianKernel(α)
+
+Probability density function of distance ``d``.
+
+```math
+y = \frac{(α+1)(α+2)}{2 π} (1+d)^α
+```
+
+where α is a positive parameter.
+"""
+@Base.kwdef struct GaussianKernel{P} <: KernelFormulation
+    α::P = Param(1.0, bounds=(0.0, 1000.0))
+end
+(f::GaussianKernel)(d) = 1/(π*f.α^2)*exp(-d^2/f.α^2)
+
+
+"""
+    BivariateStudentKernel <: KernelFormulation
+
+    BivariateStudentKernel(α, β, θ, c1, c2)
+
+Probability density function of distance ``d``.
+The Bivariate Student kernel has a power-law decrease and is anisotropic.
```

Comment:
Maybe explain it in slightly more lay terms as well, maybe give and example use-case for a directional kernel?

Reply:
ok. and the parameterisation I used is isotropic 👎